src/mn_setup.c

```diff
@@ -2115,6 +2120,9 @@ static setup_menu_t enem_settings1[] = {
     {"Translucent Ghost Monsters", S_ONOFF | S_STRICT | S_VANILLA, M_X, M_SPC,
      {"ghost_monsters"}},
 
+    {"Spectre Drawing", S_CHOICE | S_STRICT, M_X, M_SPC, {"fuzzcolumn_mode"},
```

Comment:
This applies to the effect in general (i.e. weapon too), so maybe we should use this opportunity to name this better.

Reply:
Also, it should probably be moved to General > Display.